juriscraper/AbstractSite.py

```diff
@@ -287,6 +288,32 @@ def _make_html_tree(self, text):
         """
         return get_html_parsed_text(text)
 
+    def _get_html_tree_by_url(self, url, request_dict={}):
```

Comment:
I think this is a good idea to provide this, but I think you're right that the code in this method and the code in `_download` should be generalized and pulled out. 

This might also be a good time to pull the dict out of the default keyword argument, since [it's a common cause of bugs](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments). I don't think it's causing us issues, but it's not ideal, and we may as well fix it.